x-pack/plugins/security_solution/server/cloud_posture/routes/score.ts

```diff
@@ -5,28 +5,22 @@
  * 2.0.
  */
 
-/* eslint-disable @typescript-eslint/no-explicit-any */
 /* eslint-disable  */
 
 import { ElasticsearchClient, Logger } from 'src/core/server';
-import {
-  AggregationsFiltersAggregate,
-  SearchRequest,
-  CountRequest,
-  SearchSort,
-} from '@elastic/elasticsearch/lib/api/types';
+import { SearchRequest, CountRequest } from '@elastic/elasticsearch/lib/api/types';
 import type { SecuritySolutionPluginRouter } from '../../types';
-import { any, string } from 'joi';
+import type { CloudPostureStats, PostureScore } from '../types';
 
 const FINDINGS_INDEX = `kubebeat*`;
 
-const getFindingsEsQuery = (benchmark: string = '*', runId: string): CountRequest => {
+const getFindingsEsQuery = (benchmark: string = '*', cycleId: string): CountRequest => {
```

Comment:
TS has [type inference](https://www.typescriptlang.org/docs/handbook/type-inference.html)

so we whenever we assign a value to a variable, that type of that variable will be inferred from the value

so we don't need to say `benchmark: string = '*'` because it's already inferred to be a string

Reply:
done

Reply:
```suggestion
const getFindingsEsQuery = (cycleId: string, benchmark = '*', ): CountRequest => {
```

note that currently we have `*` as default `benchmark` value but still we're calling this function with `*` because the of the arguments order.

if we move `benchmark` to be the last param, we can actually benefit from the default param